lib/features/push_notification/presentation/services/fcm_receiver.dart

```diff
@@ -26,7 +29,12 @@ class FcmReceiver {
   }
 
   void getFcmToken() async {
-    final token = await FirebaseMessaging.instance.getToken();
-    log('FcmReceiver::onFcmToken():token: $token');
+    try {
+      final token = await FirebaseMessaging.instance.getToken(vapidKey: kIsWeb ? dotenv.get('FIREBASE_WEB_VAPID_PUBLIC_KEY', fallback: '') : null);
+      log('FcmReceiver::onFcmToken():token: $token');
+    } catch(e) {
+      log('FcmReceiver::onFcmToken():exception: $e');
+      throw NotLoadedFCMTokenException();
```

Comment:
You should `try/catch` when calling this function. Avoid crashes

Reply:
it `try/catch`  in `injectFCMBindings`